.nvmrc

```diff
@@ -0,0 +1 @@
+v12.19
```

Comment:
Can probably delete this file if you're okay using [Volta](https://volta.sh/) instead of `nvm`?

Reply:
Absolutely! 👍